src/MsgProcessing.chpl

```diff
@@ -60,10 +69,16 @@ module MsgProcessing
         var rname = st.nextName();
         
         // if verbose print action
-        if v {try! writeln("%s %s %i : %s".format(cmd,dtype2str(dtype),size,rname)); try! stdout.flush();}
+        mpLogger.debug(getModuleName(),getRoutineName(),getLineNumber(), 
+            "cmd: %s dtype: %s size: %i new pdarray name: %s".format(
+                                                     cmd,dtype2str(dtype),size,rname));
         // create and add entry to symbol table
         st.addEntry(rname, size, dtype);
-        // response message
+        // if verbose print result
+        mpLogger.debug(getModuleName(),getRoutineName(),getLineNumber(), 
+                                    "created the pdarray %t".format(st.lookup(rname)));
```

Comment:
Noted in https://github.com/mhmerrill/arkouda/issues/614, but I think this is resulting in trying to convert the entire symbol entry and the array it contains into a string.